LICENSE.md

```diff
@@ -1,27 +1,61 @@
-Copyright © 2014–2021, Mapbox
+# License
+
+Mapbox Navigation for iOS version 2.0
+Mapbox Navigation iOS SDK
+
+Copyright © 2021 Mapbox
+
+All rights reserved.
+
+Mapbox Navigation for iOS version 2.0 (“Mapbox Navigation iOS SDK”) or higher must be used 
+according to the Mapbox Terms of Service. This license allows developers with a current active Mapbox account to 
+use and modify the Mapbox Navigation iOS SDK. Developers may modify the Mapbox Navigation iOS SDK 
+code so long as the modifications do not change or interfere with marked portions of the code related to billing, 
+accounting, and anonymized data collection. The Mapbox Navigation iOS SDK sends anonymized location
+and usage data, which Mapbox uses for fixing bugs and errors, accounting, and generating aggregated anonymized 
+statistics. This license terminates automatically if a user no longer has an active Mapbox account.
+
+For the full license terms, please see the Mapbox Terms of Service at https://www.mapbox.com/legal/tos/
+
+-------------------------------------------------------------------------------
 
-Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.
+## Third-party licenses
 
-THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+Mapbox Navigation iOS SDK uses adapted implementation of the `Minimum Edit Distance` algorithm in `MinimumEditDistance.swift`.  
+URL: [https://github.com/raywenderlich/swift-algorithm-club/blob/master/Minimum%20Edit%20Distance/MinimumEditDistance.playground/Contents.swift](https://github.com/raywenderlich/swift-algorithm-club/blob/master/Minimum%20Edit%20Distance/MinimumEditDistance.playground/Contents.swift).  
+License: [MIT License](https://github.com/raywenderlich/swift-algorithm-club/blob/master/LICENSE.txt).
 
-----
+Mapbox Navigation iOS SDK uses `Solar`.  
+URL: [https://github.com/ceeK/Solar](https://github.com/ceeK/Solar).  
+License: [MIT License](https://github.com/ceeK/Solar/blob/master/LICENSE).  
 
-Copyright (c) 2016 Matthijs Hollemans and contributors
```

Comment:
This became a link, but there does need to be an inline license because we’ve vendored the code rather than pulling it in via a dependency manager:

https://github.com/mapbox/mapbox-navigation-ios/blob/efbe4d129bd3ab95074de35ad5eff260901f98b5/LICENSE.md#L24-L26

Reply:
@1ec5, @kathleenlu09, shall I remove current link to `MinimumEditDistance.swift` and replace it wit plain license as it was before?

Reply:
@MaximAlien I think you can have both the link and the plain license:

>  Mapbox Navigation iOS SDK uses an adapted implementation of the `Minimum Edit Distance` algorithm in `MinimumEditDistance.swift`.   
> 
>  URL for original: [https://github.com/raywenderlich/swift-algorithm-club/blob/master/Minimum%20Edit%20Distance/MinimumEditDistance.playground/Contents.swift](https://github.com/raywenderlich/swift-algorithm-club/blob/master/Minimum%20Edit%20Distance/MinimumEditDistance.playground/Contents.swift).   
> 
>  License: [MIT License](https://github.com/raywenderlich/swift-algorithm-club/blob/master/LICENSE.txt). 
> Copyright (c) 2016 Matthijs Hollemans and contributors
> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

Reply:
Thanks, added.